adapter/src/dbt/adapters/fal/wrappers.py

```diff
@@ -94,6 +94,15 @@ def __getattr__(self, name):
         else:
             getattr(super(), name)
 
+    def get_relation(self, database: str, schema: str, identifier: str):
+        # HACK: When compiling Python models, we get an all-False quoting policy
+        #       This does not happen in 1.4
+        if self._db_adapter.type() == "athena":
+            # and dbt-athena-community breaks for that case
+            self.config.quoting = {"database": True, "schema": True, "identifier": True}
+
+        return self._db_adapter.get_relation(database, schema, identifier)
+
```

Comment:
NIT: Does it matter if we put it in a wrapper or in `impl.py`? I think `impl.py` is better because it's more visible there. This feels more like putting something under a rug.

Reply:
The way I saw this change was taking it from the "general" soluton of the Adapter Mixin and putting it in the wrapper that is actually being used, since this is 1.3-specific and will not be replicated for 1.4

Reply:
merging and you can make a decision in your PR